frontend/public/components/utils/resource-log.tsx

```diff
@@ -148,6 +159,45 @@ const showDebugAction = (pod: PodKind, containerName: string) => {
   return !containerStatus?.state?.running || !containerStatus.ready;
 };
 
+const getPodLogLinkUrl = (podLogLink, templateValues) => {
+  const { namespaceFilter, hrefTemplate } = podLogLink.spec;
+  if (namespaceFilter) {
+    try {
+      const namespaceRegExp = new RegExp(namespaceFilter, 'g');
+      if (templateValues.namespace.search(namespaceRegExp)) {
+        return '';
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.warn('invalid log link regex', namespaceFilter, e);
+      return '';
+    }
+  }
+  return replaceVariables(hrefTemplate, templateValues);
+};
+
+const getPodLogExternalLinks = (podLogLinks, templateValues) =>
```

Comment:
Can you also add some times here please. Thanks

Reply:
updated